onnxruntime/core/common/profiler.h

```diff
@@ -121,7 +121,16 @@ class Profiler {
   // Mutex controlling access to profiler data
   OrtMutex mutex_;
   bool enabled_{false};
+#if defined(__wasm__)
+  /*
+   * The simplest way to emit profiling data in WebAssembly is to print out to console,
+   * since browsers can't access to a file system directly.
+   * TODO: Consider MEMFS or IndexedDB instead of console.
+   */
+  std::ostream& profile_stream_{std::cout};
```

Comment:
is it a good idea to lazy-initialize? ( to save memory usage )

Reply:
I don't think we can do with std::cout unless we replace all profile_stream_ to std::cout explicitly, which would be ugly.

Reply:
is it possible to initialize profile_stream_ to std::cout at https://github.com/microsoft/onnxruntime/pull/8932/files#diff-d0251c9ed4b9395c8bacb3dc7d56d6421fd1e4c7ef8fd16683fc604272ad12dbR52 ?